pkg/client/cli/telepresence_test.go

```diff
@@ -1753,6 +1833,34 @@ func (ts *telepresenceSuite) capturePodLogs(ctx context.Context, app, ns string)
 	return nil
 }
 
+func annotateForWebhook(ctx context.Context, objKind, objName, objNamespace string, servicePort int) error {
+	err := run(ctx, "kubectl", "patch", "-n", objNamespace, objKind, objName, "-p", fmt.Sprintf(`
+{
+	"spec": {
+		"template": {
+			"metadata": {
+				"annotations": {
+					"telepresence.getambassador.io/inject-traffic-agent": "enabled",
+					"telepresence.getambassador.io/inject-service-port": "%d"
+				}
+			}
+		}
+	}
+}`, servicePort))
+	if err != nil {
+		return err
+	}
+
+	return run(ctx, "kubectl", "rollout", "status", "-w", fmt.Sprintf("%s/%s", objKind, objName), "-n", objNamespace)
+}
+
+func dropWebhookAnnotation(ctx context.Context, objKind, objName, objNamespace string) error {
+	return run(ctx, "kubectl", "patch", "-n", objNamespace, objKind, objName, "--type=json", "-p", `[{
+	"op": "remove",
+	"path": "/spec/template/metadata/annotations"
```

Comment:
This will remove all annotations right? Some of the yaml we use in the tests do have annotations (e.g. https://github.com/telepresenceio/telepresence/blob/release/v2/k8s/with-probes.yaml#L13-L15). Based on passing tests I'm not sure they're vital to anything, but it might be worth either:
A. making this function only drop the annotations related to the webhook 
B. Change the function name to dropAnnotations (so that it's clear that it drops all annotations instead of just the webhook-related ones.